no1=10
no2 =0
try:
    print(no1/no2)
except ZeroDivisionError:
    print("Exception thrwon")
print("After exception")